flask/flask-common/src/main/java/net/corda/flask/common/Flask.java

```diff
@@ -0,0 +1,148 @@
+package net.corda.flask.common;
+
+import lombok.SneakyThrows;
+
+import java.io.*;
+import java.math.BigInteger;
+import java.security.DigestInputStream;
+import java.security.MessageDigest;
+import java.util.AbstractMap;
+import java.util.GregorianCalendar;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.util.zip.CRC32;
+import java.util.zip.ZipEntry;
+
+public class Flask {
+
+    public static class Constants {
+        public static final String DEFAULT_LAUNCHER_NAME = "net.corda.flask.launcher.Launcher";
+        public static final String LIBRARIES_FOLDER = "LIB-INF";
+        public static final String METADATA_FOLDER = "META-INF";
+        public static final String JVM_ARGUMENT_FILE = METADATA_FOLDER + "/jvmArgs.properties";
+        public static final String JAVA_AGENTS_FILE = METADATA_FOLDER + "/javaAgents.properties";
+        public static final String CLI_JVM_PARAMETERS_PREFIX = "-flaskJvmArg=";
+        public static final long ZIP_ENTRIES_DEFAULT_TIMESTAMP =
+                new GregorianCalendar(1980, 1, 1, 0, 0, 0).getTimeInMillis();
```

Comment:
I see that `ZipCopyAction` is an internal Gradle class, so I suppose it's better not to use it directly. But can you at least add a comment to explain that this value is chosen to match Gradle's own `ZipCopyAction.CONSTANT_TIME_FOR_ZIP_ENTRIES` please? Because we need to remember where it came from.

BTW, the month value is better expressed as `Calendar.FEBRUARY`.

Reply:
👍🏿